modules/dataquery/ajax/retrieveCategoryDocs.php

```diff
@@ -10,7 +10,9 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://www.github.com/aces/Loris/
  */
-ini_set("max_input_vars", 10000);
+
+
```

Comment:
@johnsaigle If you leave 2 empty line here, you should do the same thing in modules/dataquery/ajax/saveQuery.php

Reply:
Okay I changed the other file to have new lines but I really don't think something like this is worth blocking the PR for.

Reply:
I don't know of any coding standard that suggests two blank lines, this'll just make it require another PR later to fix this. There's no reason to not fix it now.